src/run_tribler.py

```diff
@@ -29,6 +30,8 @@ def start_tribler_core(base_path, api_port, api_key, root_state_dir):
 
     trace_logger = None
 
+    asyncio.set_event_loop(asyncio.SelectorEventLoop())
```

Comment:
Why?

Reply:
Because using the `ProactorEventLoop` on Windows seems to break peer discovery in IPv8. For the moment being, and maybe even for the 7.5.2 release, I suggest to use the `SelectorEventLoop`, even though we know it has deficiencies and that this is the primary reason we moved to Python 3.8.

I leave the debugging of this behaviour/finding a bug to someone who is more experienced with IPv8 peer discovery.

Also see #5485

Reply:
Please, leave a comment in the code. And limit the change to the Windows platform.